### PR TITLE
change load fbref to releases, fix tm_staff_history error

### PR DESCRIPTION
**Pull request by JaseZiv**
`load_fb` functions now point to the data in releases.

Also fixed the `tm_staff_job_history()` bug due to HTML changes

### CODE REVIEW

File: R/load_fb.R [hunk 1]
```diff
@@ -30,7 +30,9 @@
 #' }
 
 load_match_results <- function(country, gender, season_end_year, tier) {
-  dat_urls <- paste0("https://github.com/JaseZiv/worldfootballR_data/blob/master/data/match_results/", country, "_match_results.rds?raw=true")
+  "https://github.com/JaseZiv/worldfootballR_data/releases/download/match_results/ARG_match_results.rds"
```
Comment (tonyelhabr) [inline R/load_fb.R:33]: looks like test code 😄

File: NEWS.md [hunk 2]
```diff
@@ -8,6 +8,16 @@
...
 * `fotmob_get_season_stats` internals changed to handle new data format  (0.6.1.5000)
+* `tm_staff_job_history()` code changed due to HTML changes on transfermarkt.com (0.6.1.6000)
+
+
+### Improvements
+
+* `load_fb_big5_advanced_season_stats()` now loads FBRef data provided by Stats Perform (Opta) (0.6.1.6000)
+* `load_match_results()` now loads FBRef data provided by Stats Perform (Opta) (0.6.1.6000)
+* `load_match_comp_results()` now loads FBRef data provided by Stats Perform (Opta) (0.6.1.6000)
+* `load_understat_league_shots()` now loads data from GitHub releases (0.6.1.6000)
```
Comment (tonyelhabr) [inline NEWS.md:19]: 👍

Comment (tonyelhabr) [reply NEWS.md:19]: you could say "via Github releases" with the three FBref functions so it's clear that it's both a change in data source (by FBref) and a change in data format (from us)

**Review by tonyelhabr — APPROVED** [review]

File: R/load_fb.R (hunk 1 above)
Comment (JaseZiv) [reply R/load_fb.R]: It absolutely was! Embarrassing! Thanks for picking up. Rectified now

File: NEWS.md (hunk 2 above)
Comment (JaseZiv) [reply NEWS.md]: Nice suggestion! Implemented

**Pull request merged**